prisma/schema.prisma

```diff
@@ -1,39 +1,46 @@
 generator client {
-  provider = "prisma-client-js"
+  provider      = "prisma-client-js"
   binaryTargets = ["native", "linux-musl"]
 }
 
 datasource db {
-  provider = "mysql"
-  url      = env("DATABASE_URL")
+  provider          = "mysql"
+  url               = env("DATABASE_URL")
   shadowDatabaseUrl = env("SHADOW_DATABASE_URL")
 }
 
 model PaybuttonAddress {
-  id          Int        @id @default(autoincrement())
-  address     String     @db.VarChar(255)
-  createdAt   DateTime   @default(now())
-  updatedAt   DateTime   @updatedAt
+  id          Int       @id @default(autoincrement())
+  address     String    @db.VarChar(255)
+  createdAt   DateTime  @default(now())
+  updatedAt   DateTime  @updatedAt
   chainId     Int
   paybuttonId Int
   chain       Chain     @relation(fields: [chainId], references: [id], onUpdate: Restrict)
-  paybutton  Paybutton @relation(fields: [paybuttonId], references: [id], onDelete: Cascade, onUpdate: Restrict)
+  paybutton   Paybutton @relation(fields: [paybuttonId], references: [id], onDelete: Cascade, onUpdate: Restrict)
+
+  @@index([chainId], map: "PaybuttonAddress_chainId_fkey")
+  @@index([paybuttonId], map: "PaybuttonAddress_paybuttonId_fkey")
 }
 
 model Paybutton {
-  id                  Int                   @id @default(autoincrement())
-  providerUserId      String?               @db.VarChar(255)
-  createdAt           DateTime              @default(now())
-  updatedAt           DateTime              @updatedAt
-  addresses           PaybuttonAddress[]
+  id             Int                  @id @default(autoincrement())
+  name           String               @db.VarChar(255)
+  uuid           String               @default(dbgenerated("uuid()"))
+  buttonData     String               @db.LongText
+  providerUserId String?              @db.VarChar(255)
+  createdAt      DateTime             @default(now())
+  updatedAt      DateTime             @updatedAt
+  addresses      PaybuttonAddress[]
+
+  @@unique([name, providerUserId], map: "Paybutton_name_providerUserId_unique_constraint")
 }
 
 model Chain {
-  id                  Int                   @id @default(autoincrement())
-  slug                String                @unique @db.VarChar(255)
-  title               String                @db.VarChar(255)
-  createdAt           DateTime              @default(now())
-  updatedAt           DateTime              @updatedAt
-  paybuttonAddresses  PaybuttonAddress[]
-
+  id                 Int                @id @default(autoincrement())
+  slug               String             @unique @db.VarChar(255)
```

Comment:
Is this supposed to be the currency ticker symbol / currency code?

Reply:
Yes, e.g.:
![image](https://user-images.githubusercontent.com/21281174/174416879-32156969-9131-4cb5-baad-bcd101ee27a0.png)

Reply:
Ok this is a bit different than that. Can we add a column for ticker?

`BCH`
`XEC`

(testnet would have the same tickers as mainnet).

Reply:
Yes, I'll address that in a separate PR.